Source/WebCore/platform/gamepad/cocoa/GameControllerGamepad.h

```diff
@@ -59,14 +59,15 @@ class GameControllerGamepad : public PlatformGamepad {
 
 #if HAVE(WIDE_GAMECONTROLLER_SUPPORT)
     GameControllerHapticEngines& ensureHapticEngines();
+    Ref<GameControllerHapticEngines> protectedHapticEngines() { return ensureHapticEngines(); }
```

Comment:
Should be protectedEnsureHapticEngines()

Reply:
We usually go with `ensureProtectedHapticEngines()` I believe.